### PR TITLE
Small UI fixes

### PR DESCRIPTION
**Pull request by vallmoy**

### CODE REVIEW

File: properties/world.py [hunk 1]
```diff
@@ -42,8 +42,8 @@ def register(cls):
                                                      ("gradient", "Gradient", "Use sky color gradient"),
                                                      ("latlong_map", "Latitude-Longitude Map", "Use latlong map texture"),
                                                      ("mirrorball_map", "Mirror Ball Map", "Use mirror ball texture"),
-                                                     ("sunsky", "Physical Sun/Sky", ""),
-                                                     ("constant_hemisphere", "Per-Hemisphere Constant", "Use constant color per hemisphere")],
+                                                     ("constant_hemisphere", "Per-Hemisphere Constant", "Use constant color per hemisphere"),
```
Comment (dictoon) [inline properties/world.py]: Is this going to break compatibility with existing .blend files, since you're reordering items in the combo?

If so, we need to revert this change since we pledged not to break compatibility again.

Comment (vallmoy) [reply properties/world.py]: Ok, reverted.
